Marsiple/App/Posts/PostTableViewCell.swift

```diff
@@ -40,6 +40,7 @@ private extension PostTableViewCell {
         title.textColor = .black
         title.textAlignment = .left
         title.font = .systemFont(ofSize: 20)
+        title.numberOfLines = 0
```

Comment:
I feel that `title`, since it's a `UILabel` instance, should be name more aptly - `titleLabel` sounds more descriptive and concise.